Sources/App/Utilities/OpenInFirefoxControllerSwift.swift

```diff
@@ -45,11 +45,12 @@ open class OpenInFirefoxControllerSwift {
         app.canOpenURL(URL(string: "\(type.urlScheme)://")!)
     }
 
-    open func openInFirefox(_ url: URL) {
+    open func openInFirefox(_ url: URL, privateTab: Bool = false) {
```

Comment:
```suggestion
    open func openInFirefox(_ url: URL, privateTab: Bool) {
```

since the parameter is always provided, let's drop the default value

Reply:
The `privateTab` parameter gets provided just for Firefox as derivatives (Firefox Focus and Klar) don't support private tabs and get passed just the `url`.  Is this fine to stay as is?

Reply:
Fair point - I think this can stay with a default parameter